### ISSUE TITLE
remove dependency on Apache commons-io

### ISSUE THREAD

**Issue by filfreire** [yegor256/rultor#1207]
Remove Apache commons-io dependency and replace it with Cactoos or Jcabi equivalents.

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by filfreire** [comment]
I volunteer to work on this 👍

**Comment by yegor256** [comment]
@0crat assign @filfreire

**Comment by 0crat** [comment]
> @0crat assign @filfreire  ([here](https://github.com/yegor256/rultor/issues/1207#issuecomment-323962799))

@yegor256 Job `gh:yegor256/rultor#1207` assigned to @filfreire, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)). @filfreire is not a member of this project yet, but can request to join. @filfreire check your [Zerocracy profile](http://www.0crat.com/u/filfreire) and follow the instructions.

**Comment by 0crat** [comment]
@filfreire this job was assigned to you 8 days ago.
It will be taken away from you after 10 days from start (this is our policy).

**Comment by 0crat** [comment]
@filfreire resigned from `gh:yegor256/rultor#1207`, please stop working.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/rultor#1207` is not assigned to anyone.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/rultor#1207` is not assigned to anyone.

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/rultor#1207` is not assigned to anyone.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by yegor256** [comment]
@filfreire do you want to finish this or we should remove it from scope?

**Comment by filfreire** [comment]
@yegor256, I want to do it. Please don't remove it from scope.

**Comment by yegor256** [comment]
@0crat assign @filfreire

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by yegor256** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat assign @filfreire  ([here](https://github.com/yegor256/rultor/issues/1207#issuecomment-383656187))

@yegor256 The job [#1207](https://github.com/yegor256/rultor/issues/1207) assigned to @filfreire[/z](https://www.0crat.com/u/filfreire), here is [why](https://www.0crat.com/footprint/C3SAYRPH9/1301336350); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/rultor/issues/1207#issuecomment-388054983))

@yegor256 @filfreire[/z](https://www.0crat.com/u/filfreire) resigned from [#1207](https://github.com/yegor256/rultor/issues/1207), since the job is not in scope anymore

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/rultor/issues/1207#issuecomment-388054983))

@yegor256 The job [#1207](https://github.com/yegor256/rultor/issues/1207) is now out of scope

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
It is strongly discouraged to assign jobs to their creators, see [§19](http://www.zerocracy.com/policy.html#19): -15 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by pnatashap** [comment]
According to the dependency tree commons-io is needed only for 
```
[INFO] +- com.jcabi:jcabi-s3:jar:0.19.0:compile
[INFO] |  \- commons-io:commons-io:jar:2.15.1:compile
```
it can be deleted only after deleting from jcabi-s3

**Comment by yegor256** [comment]
@pnatashap let's take it off jcabi-s3 too?

**Comment by 0pdd** [comment]
@filfreire the puzzle [#1918](https://github.com/yegor256/rultor/issues/1918) is still not solved.

**Comment by 0pdd** [comment]
@filfreire the only puzzle [#1918](https://github.com/yegor256/rultor/issues/1918) is solved here.
